### ISSUE TITLE
Focus and Mozilla

### ISSUE THREAD

**Issue by OniMeka** [onisep/ibexa-imagemap-bundle#3]
Hi 

The focus of the "Area" Image Map element does not have the same blue border on Mozilla as on Chrome or Opera. The border on Mozilla isn't enough visible, expecialy during a keyboard navigation.
